widgets/tabs/styles/default.less

```diff
@@ -113,4 +131,26 @@
 			}
 		}
 	}
+
+	& when ( @tabs_container_tabs_position = bottom ) {
+		& {
+			display: flex;
+		    flex-direction: column-reverse;
+		}
+	}
+
+	& when ( @tabs_container_tabs_position = left ), ( @tabs_container_tabs_position = right ) {
+		display: flex;
```

Comment:
It seems like all options are using `display: flex;`, in which case it only needs to be set once.

Reply:
Top doesn't. Regardless, I've used `when not` to apply this when Top isn't being used.